From 440c3c39c51c89c0a5bc53850e9b75b142a39bd2 Mon Sep 17 00:00:00 2001
From: Michael Vetter <jubalh@iodoru.org>
Date: Wed, 14 Feb 2018 15:02:22 +0100
Subject: [PATCH] Use fputs instead of fput

fput() seems to be non-standard.
See commit cb0aadd0a2f16a09bc972d648b864a1fd541e75f
Now we have the original patch.
---
 lpost.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lpost.c b/lpost.c
index a2c0d29..39bb17e 100644
--- a/lpost.c
+++ b/lpost.c
@@ -42,7 +42,7 @@ int main(int argc,char *argv[]) {
 			}
 		}
 		else {
-			(void) fput(line, pfp);
+			(void) fputs(line, pfp);
 		}
 	} /* end while */
 	exit(retval);
